processing/data_collection/gazette/spiders/sc_joinville.py

```diff
@@ -0,0 +1,53 @@
+import dateparser
```

Comment:
This file may not be here. It is not related to your PR that references only Teresina-PI spider.